command_line/formatting.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
```
cli
- __init__.py
- format.py (= formatting.py)
- main.py (= pyreportcard-cli.py)
```

I think it is better than

```
command_line
- __init__.py
- formatting.py
pyreportcard-cli.py
```

And in later, we can make a `pyreportcard` command line using `cli`.

Reply:
`format` is also built-in, I'd like to avoid using that name which is why I used `formatting` but I'm OK about the other changes.
https://docs.python.org/3/library/functions.html#format

Reply:
@xavierhardy Ah, so `formatting` also looks good.